package.json

```diff
@@ -156,6 +156,7 @@
     }
   },
   "dependencies": {
+    "@apollo/client": "3.5.9",
```

Comment:
usually, we don't add global packages. It's better to add this package locally to your app/package

Reply:
fixed